apollo-client/src/test/java/com/ctrip/framework/apollo/spring/XmlConfigPlaceholderTest.java

```diff
@@ -133,7 +128,40 @@ public void testWithInvalidWeight() throws Exception {
     check("spring/XmlConfigPlaceholderTest5.xml", DEFAULT_TIMEOUT, DEFAULT_BATCH);
   }
 
-  private void check(String xmlLocation, int expectedTimeout, int expectedBatch) {
+
+  @Test
+  public void testResolveNamespacesWithDefaultValue() throws Exception {
```

Comment:
this case failed in my machine, as there is no value for xxx.from.system.property

Reply:
btw, this case looks like the same as `testUnresolvedNamespaces`

Reply:
Fix it in commit https://github.com/ctripcorp/apollo/pull/3349/commits/b5eb6f0b3f77b2b62441b2f77b7fad8873704f88